docs/conf/vilnius/2019/news/volunteers-cfp-reminder.rst

```diff
@@ -0,0 +1,50 @@
+:template: {{year}}/generic.html
+
+.. post:: April 3, 2019
+   :tags: vilnius-2019, volunteers, cfp, tickets
+
+Call for Volunteers and CFP Reminder
+====================================
+
+Greetings, fellow documentarians! The Vilnius conference just about two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
+
+**Tickets are selling fast!** We are not sold out yet, but our European community is showing great enthusiasm, and we expect to have another wave of registrations after we announce the presentations soon, so you might want to beat the rush and `get your ticket now <https://ti.to/writethedocs/write-the-docs-vilnius-2019>`_.
+
+Call for volunteers
+-------------------
+
+There is plenty to do during the conference, and volunteers are essential to the success of all our events! We are looking for folks who are willing to help out during the conference. If you volunteer for two or more shifts (at an average of three hours each), you will receive a free ticket for the conference.
```

Comment:
Still 2 or more even though we're only having 1 day and 1 writing day?

Reply:
Yup, I think that's not a problem in this case